app/views/metrics/index.html.erb

```diff
@@ -41,65 +34,20 @@
       </div>
     </div>
     <div class="thirteen wide column">
-      <% if @defined_metrics then %>
-        <div class="ui tab segments active" data-tab="new_tab">
-          <% if @new_students > 0 then %>
-            <!-- TODO: Remove rendering -->
-            <%= render partial: 'student_row', locals: {student_name: "Jane Doe", grace_days: 3, low_scores: 2} %>
-            <%= render partial: 'student_row', locals: {student_name: "Emily Lee", missing_submissions: 2} %>
-            <%= render partial: 'student_row', locals: {student_name: "John Smith", downward_trend: 1} %>
-          <% else %>
-            <div id="undefined_metrics">
-              <center>
-                <i class="huge inbox icon"></i> 
-                <h3> There are no new students at risk </h3>
-              </center>
-            </div>
-          <% end %>
-        </div>
-        <div class="ui tab segments" data-tab="contacted_tab">
-          <% if @contacted_students > 0 then %>
-          <% else %>
-            <div id="undefined_metrics">
-              <center>
-                <i class="huge inbox icon"></i> 
-                <h3> You have not contacted any students </h3>
-              </center>
-            </div>
-          <% end %>
-        </div>
-        <div class="ui tab segments" data-tab="resolved_tab">
-          <% if @resolved_students > 0 then %>
-          <% else %>
-            <div id="undefined_metrics">
-              <center>
-                <i class="huge inbox icon"></i> 
-                <h3> You have not resolved any students </h3>
-              </center>
-            </div>
-          <% end %>
-        </div>
-        <div class="ui tab segments" data-tab="archived_tab">
-          <% if @archived_students > 0 then %>
-          <% else %>
-            <div id="undefined_metrics">
-              <center>
-                <i class="huge inbox icon"></i> 
-                <h3> You have not archived any students </h3>
-              </center>
-            </div>
-          <% end %>
-        </div>
-      <% else %>
-        <div class="ui tab segments active" id="undefined_metrics">
-          <center>
-            <i class="huge inbox icon"></i> 
+      <div id="defined_metrics">
+        <div class="ui tab segments active" data-tab="new_tab" id="new_tab"></div>
+        <div class="ui tab segments" data-tab="contacted_tab", id="contacted_tab"></div>
+        <div class="ui tab segments" data-tab="resolved_tab", id="resolved_tab"></div>
+        <div class="ui tab segments" data-tab="archived_tab", id="archived_tab"></div>
+      </div>
+      <div class="ui tab segments active" id="undefined_metrics">
+        <center>
+          <i class="huge inbox icon"></i> 
             <h3> You have not defined your risk metrics </h3>
             Set up identification conditions at in the <b>Risk Metrics</b> tab. <br> 
             Identified possible at-risk students will appear here. 
-          </center>
```

Comment:
I feel that it should not default to no metrics state, because it will be shown while the api is being called, confusing the user.

Reply:
Made this hidden by default with `display:none`

Reply:
no longer displays! :)